services/configuration/src/config_api.rs

```diff
@@ -173,50 +173,43 @@ impl PublicConfigApi {
                 .map(|(cfg_hash, propose_data)| {
                     ApiResponseProposeHashInfo {
                         hash: cfg_hash,
-                        propose_data: propose_data,
+                        propose_data,
                     }
                 })
                 .collect::<Vec<_>>()
         };
-        Ok(proposes)
+        proposes
     }
 
     fn get_all_committed(
         &self,
         previous_cfg_hash_filter: Option<Hash>,
         actual_from_filter: Option<Height>,
-    ) -> Result<Vec<ApiResponseConfigHashInfo>, ApiError> {
+    ) -> Vec<ApiResponseConfigHashInfo> {
         let snapshot = self.blockchain.snapshot();
         let general_schema = Schema::new(&snapshot);
         let index = general_schema.configs_actual_from();
         let committed_configs = {
             index
                 .into_iter()
                 .map(|reference| {
-
-                    let cfg_hash = reference.cfg_hash();
-                    let cfg = general_schema.configs().get(cfg_hash).expect(&format!(
+                    let config_hash = reference.cfg_hash();
+                    general_schema.configs().get(config_hash).expect(&format!(
```

Comment:
Does calling `configs()` method repeatedly introduce any sort of overhead? Even if not, initializing the configs table once would look more readable to me.